tests/conftest.py

```diff
@@ -16,42 +16,36 @@
 from cashu.mint import migrations as migrations_mint
 from cashu.mint.ledger import Ledger
 
-SERVER_ENDPOINT = "http://localhost:3337"
+SERVER_PORT = 3337
+SERVER_ENDPOINT = f"http://localhost:{SERVER_PORT}"
+
+settings.cashu_dir = "./test_data/"
+settings.mint_host = "localhost"
+settings.mint_port = SERVER_PORT
+settings.mint_host = "0.0.0.0"
+settings.mint_listen_port = SERVER_PORT
+settings.mint_url = SERVER_ENDPOINT
+settings.lightning = False
+settings.tor = False
+settings.mint_lightning_backend = "FakeWallet"
+settings.mint_database = "./test_data/test_mint"
+settings.mint_derivation_path = "0/0/0/0"
+settings.mint_private_key = "TEST_PRIVATE_KEY"
```

Comment:
Hrm, overwriting the settings like this strikes me as a bad idea -- to me intuitively settings should be immutable, avoids having value A at some time, then a bit of code runs, then it's now value B, things break or are confusing. Or potentially if you just import this file, it'll mangle the settings because the mutation doesn't happen in a function. Also as they're currently configured, they'll be type-checked on load (and will error if a setting is wrong), but they won't be when you re-assign them like this.

Instead it'd be 'better' to use a separate `.env` for the tests ([docs](https://docs.pydantic.dev/2.1/usage/pydantic_settings/#dotenv-env-support))

Reply:
its a hard override on settings for conftest. i think its fine, mixing up different `.env` is error prone too. i could just prefix the poetry run command inside `Makefile` but then running it with `poetry run pytest`  does not work.